bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonDeviceHandler.java

```diff
@@ -373,7 +373,9 @@ public void initialize() {
                 });
 
                 if (ThingStatus.ONLINE == bridge.getStatus()) {
-                    updateStatus(ThingStatus.ONLINE);
+                    if (ThingStatus.OFFLINE != getThing().getStatus()) {
```

Comment:
I guess test should be against ONLINE and not OFFLINE.

Reply:
No, it needs to be OFFLINE since the method `deviceNotLinked()` below could get called during initialization above if the device does not exist in the plm/hub database. Another option would be to set a flag and check that instead.